server/routes/createPlan/checkYourAnswers/checkYourAnswersController.test.ts

```diff
@@ -80,31 +102,44 @@ describe('CheckYourAnswersController', () => {
     })
 
     it('On error - Calls next with error', async () => {
+      // Given
       mockService.createCiagPlan.mockImplementation(() => {
         throw new Error('mock_error')
       })
 
-      controller.post(req, res, next)
+      // When
+      await controller.post(req, res, next)
 
+      // Then
       expect(next).toHaveBeenCalledTimes(1)
       expect(res.render).toHaveBeenCalledTimes(0)
     })
 
     it('On success - UPDATE - Calls api update', async () => {
```

Comment:
These are the tests we are interested in - the changes I've made to the `post` method, and in particular the change to `post` when its a new induction (rather than editing an existing one)